docs/code_coverage.md

```diff
@@ -2,11 +2,20 @@
 
 ## Prerequisites
 
-You need some additional Python modules to run the unit tests and linting.
+* Install all `TorchServe` prerequisites. For detailed steps refer [TorchServe installation documentation](../README.md#install-torchserve).
+
+* Install additional Python modules to run the unit tests and linting using following commands
+
+```bash
+cd <path_to_serve>
+pip install -r ci/requirements.txt
+```
+
+
+## Run sanity script
 
 ```bash
-pip install mock pytest pylint pytest-mock pytest-cov
-cd serve
+cd <path_to_serve>
```

Comment:
Line #10 above is already doing cd ... do we need this again?

Reply:
removed.